bin/appliance_console

```diff
@@ -394,6 +394,14 @@ Static Network Configuration
         task_params = []
         uri = nil
 
+        connections = ManageIQ::ApplianceConsole::Utilities.db_connections - 1
+        if connections > 0
+          say("\nDatabase restore failed")
+          say("\nThere #{connections > 1 ? "are #{connections} connections" : "is 1 connection"} preventing a database restore") if connections > 0
```

Comment:
why the if condition?... you already know the number of connections is > 0 because of line 398